Makefile

```diff
@@ -10,7 +10,7 @@ update-code:
 	./checkout-branches-for-ci.py
 
 count-todos:
-	ls *.asciidoc | xargs grep -c TODO | sed  s/:/\\t/
+	ls *.asciidoc | xargs grep -c TODO | sed  s/:/"   "/
```

Comment:
did some editor plugin do this automatically lol?  i use the tabs to be able to paste terminal output into google sheets...

Reply:
Oops.  I did that on purpose and committed it by mistake. Apparently sed handles `\t` on different platforms, so it wasn't working on my Mac.